### PR TITLE
feat: ✨ Accept argument flags for `bundle-audit check`

### PR DESCRIPTION
**Pull request by puria**
# feature

## Description

Allows to add arguments flags to the `bundle-audit check` in my case needed to ignore some CVEs

## Why should this be added

Why you should not ;)

## Checklist

- [x] My code tries to follow the style guidelines of this project

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -1,11 +1,20 @@
 name: 'Bundler Audit Action'
 description: 'GitHub Action for running bundler-audit'
 author: 'Andrew Mason <andrewmcodes@protonmail.com>'
+inputs:
+  ba_flags:
```
Comment (andrewmcodes) [inline action.yml]: What if we changed this to `options`?

**Review by andrewmcodes — CHANGES_REQUESTED** [review]
Looks good - one thought.

File: action.yml (hunk 1 above)
Comment (puria) [reply action.yml]: make sense, do you want me to do it?

File: README.md [hunk 2]
```diff
@@ -26,8 +26,9 @@ jobs:
       - uses: actions/checkout@v2
       - name: 'Bundler Audit'
         uses: andrewmcodes/bundler-audit-action@main
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          options: '--ignore CVE-2015-9284'
```
Comment (andrewmcodes) [inline README.md]: excellent! Thank you! Hopefully it makes sense why I preferred this name but I would be happy to elaborate if not!

**Review by andrewmcodes — APPROVED** [review]
Approving as this is great and works! 🎊 

---

I have one more small request though that I can implement myself if needed. I basically want to indicate on the docs that the `opts` are not required and not needed unless you need them for something custom. I personally prefer to leave a simple and advanced example here so the majority of folks don't have something they don't need in their config, and the power users know what is available. When I maintained a much larger action, the more options we added directly lead to big increases in our issue counts because the more options you need, chances are the more funky your local setup is. Providing minimal examples will help us avoid debugging someone's app only to determine this action isn't the issue. 

I just threw something together but would love to see what you come up with. Alternatively, some light editing and `copy+paste` will work as well.

Thanks for contributing!

--------

## Basic Usage

Equivalent to running `gem install bundle-audit && bundle-audit check` on your local machine:

```yaml
name: Bundler Audit

on: [push]

jobs:
  audit:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - name: 'Bundler Audit'
        uses: andrewmcodes/bundler-audit-action@main
        with:
          github_token: ${{ secrets.GITHUB_TOKEN }}
```

## Advanced Usage

The `options` argument exists to allow you to pass options to the audit command, just like you would locally:

```yaml
name: Bundler Audit

on: [push]

jobs:
  audit:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - name: 'Bundler Audit'
        uses: andrewmcodes/bundler-audit-action@main
        with:
          github_token: ${{ secrets.GITHUB_TOKEN }}
          options: '--ignore CVE-2015-9284 --format json'
```

View [rubysec/bundler-audit](https://github.com/rubysec/bundler-audit#synopsis) for documentation on what options are available.

> Note: bundler-audit also supports a per-project configuration file: `.bundle-audit.yml` which is NOT supported by this action. This action is as simple as it can be to help you get started quickly and is not meant for long term use. In order to dissuade long term use, config file support will not be implemented, or accepted via PR, without a persuading argument as to why. Thanks for understanding!

**Comment by knightq** [comment]
Any chance to see this merged? ☺️
Thank you!

**Review by knightq — APPROVED** [review]
Looks awesome to me! 🚀

**Comment by knightq** [comment]
Keep in mind that you can have the same result without this change by adding a `.bundler-audit.yml` file with the list of ignored CSV in it as described [here](https://github.com/rubysec/bundler-audit#configuration-file).

**Comment by puria** [comment]
> Keep in mind that you can have the same result without this change by adding a `.bundler-audit.yml` file with the list of ignored CSV in it as described [here](https://github.com/rubysec/bundler-audit#configuration-file).

Yeah actually I can close this, I'm using my fork I was waiting for the merge, but two years already gone, seems not gonna happen ;)

**Pull request closed without merging**